elide-standalone/src/test/java/example/ElideStandaloneExportTest.java

```diff
@@ -0,0 +1,292 @@
+/*
+ * Copyright 2018, Oath Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package example;
+
+import static io.restassured.RestAssured.when;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.yahoo.elide.ElideSettings;
+import com.yahoo.elide.ElideSettingsBuilder;
+import com.yahoo.elide.core.datastore.DataStore;
+import com.yahoo.elide.core.dictionary.EntityDictionary;
+import com.yahoo.elide.core.filter.dialect.RSQLFilterDialect;
+import com.yahoo.elide.datastores.aggregation.queryengines.sql.dialects.SQLDialectFactory;
+import com.yahoo.elide.jsonapi.links.DefaultJSONApiLinks;
+import com.yahoo.elide.standalone.ElideStandalone;
+import com.yahoo.elide.standalone.config.ElideStandaloneAnalyticSettings;
+import com.yahoo.elide.standalone.config.ElideStandaloneAsyncSettings;
+import com.yahoo.elide.standalone.config.ElideStandaloneSettings;
+import example.models.Post;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import java.util.Properties;
+import java.util.TimeZone;
+
+/**
+ * Tests ElideStandalone starts and works.
+ */
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ElideStandaloneExportTest {
+    protected ElideStandalone elide;
+
+    @BeforeAll
+    public void init() throws Exception {
+        elide = new ElideStandalone(new ElideStandaloneSettings() {
```

Comment:
Most of this is boilerplate.  Let's make a new test class that implements ElideStandaloneSettings (ElideStandaloneTestSettings) - that we can parameterized and remove this boilerplate.